src/app/portlets/content-types/fields/util/field-util.ts

```diff
@@ -16,7 +16,11 @@ export class FieldUtil {
      * @memberof ContentTypeFieldsDropZoneComponent
      */
     static isNewField(field: Field): Boolean {
-        return !field.id && !(this.isRow(field) || this.isColumn(field));
+        return !field.id;
+    }
+
+    static isNotRowOrColumn(field: Field) {
```

Comment:
I think is better to have a ```isRowOrColumn()``` and use it with ```!isRowOrColumn()``` because I think it's probably we can use ```isRowOrColumn()``` in the future, no?